src/vroom_write.cc

```diff
@@ -381,7 +381,7 @@ void vroom_write_out(
       auto end = begin + num_lines;
       futures[idx][t++] = std::async(
           fill_buf,
-          input,
+          std::ref(input),
```

Comment:
Thanks, Davis! This has a great ratio of "size of fix" to "size of headache / anxiety about problem" 😅